packages/appflowy_editor_plugins/lib/src/code_block/code_block_style.dart

```diff
@@ -1,11 +1,20 @@
 import 'package:flutter/material.dart';
 
 class CodeBlockStyle {
-  CodeBlockStyle({
-    required this.backgroundColor,
-    required this.foregroundColor,
-  });
+  const CodeBlockStyle({
+    this.textStyle,
+    this.backgroundColor,
+    this.foregroundColor,
+    this.showLineNumbers = false,
+    this.wrapLines = false,
+  }) : assert(
+          !(wrapLines && showLineNumbers),
+          'showing line numbers while wrapping isn\'t currently supported',
+        );
 
-  final Color backgroundColor;
-  final Color foregroundColor;
+  final TextStyle? textStyle;
+  final Color? backgroundColor;
+  final Color? foregroundColor;
+  final bool showLineNumbers;
```

Comment:
Maybe document comment saying that wrapLines and showLineNumbers can't both be true, and what the behavior is if both is true etc.